### PR TITLE
creates clinical assessment form

### PR DESCRIPTION
**Pull request by semujju**
WIP

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 1]
```diff
@@ -0,0 +1,1378 @@
...
+                    location.href = window.location.protocol + "//" + window.location.host +
+                        '/' + OPENMRS_CONTEXT_PATH + '/coreapps/clinicianfacing/patient.page?patientId='
+                        + queryParameters['patientId']; // Causes page to reload
+                });
+            </script>
+        </includeIf>
+    </ifMode>
+    <style type="text/css">
+
+        form#htmlform div.ai-page-frame table tbody td {
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: @semujju move these classes to the global css file

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 2]
```diff
@@ -0,0 +1,1378 @@
...
+        }
+
+        .stability {
+            display: none;
+        }
+
+    </style>
+    <!-- jquery loaded for nonessential purposes. -->
+    <script type="text/javascript">
+        function enable_disable(field, class_name_prefix, conditions, input_type) {
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: @semujju move this JS function to the global javascript file

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 3]
```diff
@@ -0,0 +1,1378 @@
...
+
+    <!-- This section contains lookup tags that pick data from other forms-->
+
+    <lookup complexExpression="#set($art_start_regimen='null') #set($other_art_start_regimen='null') #set($art_transfer_regimen='null') #set($other_art_transfer_regimen='null')"/>
+
+    <lookup complexExpression="#set($first_second_line_regimen='null') #set($other_first_second_line_regimen='null')"/>
+
+    <lookup complexExpression="#set($third_line_regimen='null') #set($other_third_line_regimen='null')"/>
+
+    <span style="display: none" id="artStartRegimen"><lookup
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml:479]: @semujju replace all display:none with boostrap class

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 4]
```diff
@@ -0,0 +1,1378 @@
...
+            complexExpression="#foreach($other_first_second_line_regimen in $fn.allObs(99146)) $other_first_second_line_regimen.valueText #end"/></span>
+
+
+    <span style="display: none" id="thirdLineRegimen"><lookup
+            complexExpression="#foreach($third_line_regimen in $fn.allObs(162990)) $first_second_line_regimen.valueCoded #end"/></span>
+
+    <span style="display: none" id="otherThirdLineRegimen"><lookup
+            complexExpression="#foreach($other_third_line_regimen in $fn.allObs(162989)) $other_first_second_line_regimen.valueText #end"/></span>
+
+    <!--End of Lookup section-->
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: @semujju delete all unused code

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 5]
```diff
@@ -0,0 +1,1378 @@
...
+                    <li class="nav-item">
+                        <a class="nav-link" data-toggle="tab" href="#investigations"> Investigations </a>
+                    </li>
+
+                    <li class="nav-item">
+                        <a class="nav-link" data-toggle="tab" href="#reference"> Reference </a>
+                    </li>
+                </ul>
+        </div>
+            <div class="card-body">
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: @semujju each tab content needs to be placed in a section as that is needed by HTML Form Entry module

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml [hunk 6]
```diff
@@ -0,0 +1,1378 @@
...
+                                            </div>
+                                        </div>
+                                    </li>
+                                </excludeIf>
+                               
+                            </ul>
+                        
+                    </div>
+
+                    <div class="tab-pane container p-5 fade" id="diagnosis">
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: @semujju why the classes container and p-5?

Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: The containers adds padding on the left and right, then the p-5 allows for top and bottom padding.

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml (hunk 1 above)
Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: ok

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml (hunk 2 above)
Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: ok

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml (hunk 4 above)
Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: ok

File: omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml (hunk 5 above)
Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml]: Noted

**Pull request closed without merging**